server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -281,6 +281,8 @@ public void accept(Object result, Throwable t) {
 
     private class MessageHandler extends SimpleChannelInboundHandler<ByteBuf> {
 
+        private ConnectionProperties connectionProperties;
+
```

Comment:
Any reason this isn't accessing `authContext.connectionProperties()` instead?

Reply:
Of course, thx!

Reply:
authContext is set to `null` in `finally` clause of `finishAuthentication`, I think this is to make sure that Credentials is garbage collected.

Reply:
But that's fine, no? The `newSession` call happens before, so by the time we need the connectionProperties the `authContext` is still available.

Reply:
yep, I was mixing things up with the sequence of calls, apologies, thx!